Sources/BuildSystemIntegration/SwiftPMBuildSystem.swift

```diff
@@ -303,7 +303,9 @@ package actor SwiftPMBuildSystem: BuiltInBuildSystem {
       swiftSDKSelector: options.swiftPMOrDefault.swiftSDK,
       store: SwiftSDKBundleStore(
         swiftSDKsDirectory: localFileSystem.getSharedSwiftSDKsDirectory(
-          explicitDirectory: options.swiftPMOrDefault.swiftSDKsDirectory.map { try AbsolutePath(validating: $0) }
+          explicitDirectory: options.swiftPMOrDefault.swiftSDKsDirectory.map {
+            try AbsolutePath(validating: $0, relativeTo: AbsolutePath(projectRoot))
+          }
```

Comment:
Does it make sense to have a SwiftSDK installed relative to your project? Don’t you typically install Swift SDKs globally?

Reply:
Yeah, it would be typically an absolute path. But I think it's not harmful to interpret it as relative to project root only when it's a relative path. Note that [`AbsolutePath(validating: relativeTo:)`](https://github.com/swiftlang/swift-tools-support-core/blob/4074f4db0971328c441fc1621c673937b9ca3b08/Sources/TSCBasic/Path.swift#L62-L64) uses the first argument as is if it's absolute.

Reply:
The thing I’m wondering is whether interpreting the path relative to the project root is the right way to go. You could have a global SourceKit-LSP configuration file at `~/.sourcekit-lsp/config.json` and it wouldn’t be unreasonable to expect that the Swift SDK path is interpreted relative to that configuration file. If we don’t allow relative paths, we don’t need to worry about any ambiguity here. 

I think for the scratch directory it’s natural to assume that it’s relative to the project root because having a global scratch directory for all project makes very little sense, so making that path relative to the project root makes perfect sense, I think.

Reply:
Ahh, that makes sense. Thank you for your explanation, I reverted swiftSDKsDirectory change.